docs/platform/howto/create-billing-groups.rst

```diff
@@ -0,0 +1,36 @@
+Create billing groups
+=====================
+
+With billing groups you can set up billing profiles to be used across multiple projects. A consolidated invoice is created for each billing group. To learn more about the key benefits of billing groups read the :doc:`related documentation <../concepts/billing-groups>`.
+
+
+Billing groups and Aiven accounts
+---------------------------------
+
+An Aiven **Account** is the entry point for creating and managing billing groups:
+
+- Within an account you can create one or more billing groups.
+- You can assign an account's projects to a billing group and move projects from one billing group to another.
+
+Aiven credits are available for all projects associated with the account. You can choose to apply the credits to a specific project within the account.
+
+Create a new billing group
+--------------------------
+
+#. In the account that you want to add a billing group to, click **Billing**.
+
+#. Click **Create billing group**.
```

Comment:
How about "select" instead of "click"? (this comment applies to the whole PR)

Reply:
I won't update each of these here since "click" is used extensively throughout the docs and I'd like us to set a firm guideline before changing something (so I don't have to change it back later :)). But, I agree this is something we should decide and include as part of the style guide!